File: src/main/java/com/jason/Hello.java
package com.jason;


import com.jason.kotlin.Student;

public class Hello {
    public static void main(String[] args) {
        Student student = new Student("Jason", 60, 70);
//        student.h
//        System.out.println("Hello world");
        String  s = "abcde";
        System.out.println(s.charAt(3));
        System.out.println(s.substring(1,4));

        Person p = new Person("Jason",66.5f, 1.7f);
        p.hello();
        System.out.println(p.bmi());
        int score = 88;
        System.out.println(score > 80 && score < 90);
        char c = 'A';
        System.out.println(c > 'a');
        /*int age = 19;
        Integer age2 = 19;
        char c = 'A';
        byte b = 120;
        float weight = 66.5f;
        float height = 1.7f;*/
    }
}

File: src/main/kotlin/com/jason/Hello.kt
package com.jason

import com.jason.kotlin.Student

fun main() {
    val student = Student("Jason", 50, 50)
    student.grading()
//    println("Hello kotlin")
    var s:String?  =  "abcde"
    s = null
    println(s?.length)
    println(s?.get(3))
    println(s?.substring(3))
    val h = Human(weight = 66.5f, height = 1.7f)
    h.hello()
    println(h.bmi())
    val score = 88
    println(score > 60)
    val c : Char = 'A'
    println(c.toInt() > 60)
    /*var age = 19
    var name: String = "Jason"
    println("$name:$age")*/
}

class Human(var name : String = "", var weight :  Float, var height : Float){
    init {
        println("test $weight")
    }
//    constructor(name: String, weight: Float, height: Float) : this(weight, height)
    fun bmi() : Float {
        val bmi = weight / (height * height)
        return bmi
}
    fun hello() {
        println("Hello kotlin")
    }
}